src/languages/internal/languagesconfiguration.cpp

```diff
@@ -33,13 +33,14 @@ using namespace mu;
 using namespace mu::framework;
 using namespace mu::languages;
 
+static const QString SYSTEM_LANGUAGE_CODE("system");
 static const std::string module_name("languages");
 static const Settings::Key LANGUAGES_JSON(module_name, "languages/languagesJson");
 static const Settings::Key LANGUAGE("ui", "ui/application/language");
 
 void LanguagesConfiguration::init()
 {
-    settings()->setDefaultValue(LANGUAGE, Val("system"));
+    settings()->setDefaultValue(LANGUAGE, Val(SYSTEM_LANGUAGE_CODE.toStdString()));
     settings()->valueChanged(LANGUAGE).onReceive(nullptr, [this](const Val& val) {
         m_currentLanguageCodeChanged.send(val.toQString());
```

Comment:
conversion is also needed here
"system" will come here if the user resets the settings to the factory

Reply:
fixed, thanks